packages/frontend/src/components/radio_button.tsx

```diff
@@ -0,0 +1,52 @@
+// RadioButton.tsx
+import type { ReactNode } from "react";
+
+import React from "react";
```

Comment:
`ReactNode` is not used, see [the failing check](https://github.com/ESP-Number-One/MatchyoSports/actions/runs/7976717417/job/21777924312#step:3:86). However it seems you use `React.ReactNode`, so you can instead remove the `React` import and remove replace `React.ReactNode` with `ReactNode` which I find more readable.

Note you will have to import `ChangeEvent`.